src/DocumentFormat.OpenXml/Office2013/Word/Word.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+
+namespace DocumentFormat.OpenXml.Office2013.Word
+{
+    public partial class Person : OpenXmlCompositeElement
+    {
+        /// <summary>
+        /// Gets or sets the <see cref="Contact"/> value. It is only available for backwards compatibility
+        /// until the next major version change. Please discontinue using this property.
+        /// </summary>
+        [Obsolete("This property will be removed in a future version")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public StringValue? Contact
+        {
+            get => GetAttribute<StringValue>();
```

Comment:
This won't work and will throw. Better to make it an auto property and update the text to say it will not persist anything to the file